tests/fxl-page-spread_combined/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:page-spread-right and -left are applied to adjacent spine items, they should be displayed in a synthetic spread.</dc:description>
+    <dc:identifier id="pub-id">fxl-page-spread_combined</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:page-spread- combined</dc:title>
```

Comment:
Unlike the page-spread-center test, -combined isn't actually a setting, so it shouldn't be hyphenated. "rendition:page-spread combined"? Or some other wording. Applies to all files in this test.